ui/job-view/headerbars/TierIndicator.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ButtonGroup, Button } from 'reactstrap';
+
+const TIERS = ['1', '2', '3'];
+
+export default function TierIndicator(props) {
+  const { filterModel } = props;
+  const shownTiers = filterModel.urlParams.tier || [];
+
+  return (
+    <ButtonGroup>
+      {TIERS.map(tier => {
+        const isOnlyTier = shownTiers.length === 1 && tier === shownTiers[0];
+        return (
+          <Button
+            onClick={() => filterModel.toggleFilter('tier', tier)}
+            title={
```

Comment:
This `title` should say something to the effect of "Toggle tier ${tier} jobs" so that the user knows what the button is for while hovering.  You can also add the line for `isOnlyTier` if it is the only `tier` showing.